src/skills-builder/skills-builder-header/messages.js

```diff
@@ -11,9 +11,14 @@ const messages = defineMessages({
     defaultMessage: 'Let edX be your guide',
     description: 'Subheading to the Skills Builder title in the header component',
   },
-  skillsBuilderHeaderTitleIsMedium: {
+  skillsBuilderHeaderEdxIsMedium: {
     id: 'skills.builder.header.title.is.medium',
-    defaultMessage: 'edX Skills builder',
+    defaultMessage: 'edX',
+    description: 'Conpany Identifier for the Skills Builder feature when screen size is medium or less',
```

Comment:
Company Identifier

Reply:
One other question is, when would `skillsBuilderHeaderEdxIsMedium` or `skillsBuilderHeaderTitleNameIsMedium` be implemented, or is that happening in a separate PR?

Reply:
Related comment, we'll need to make sure the `id` of each of these messages is different. Currently they have the same id: `skills.builder.header.title.is.medium`

Reply:
To address Jason's comment, I don't believe we need the `skillsBuilderHeaderEdxIsMedium` anymore as we are using an "edX" logo image instead of the text for the smaller views

Reply:
removed the SKillsBuilderHeaderEdxIsMedium